pyhoma/client.py

```diff
@@ -196,6 +200,12 @@ async def __post(
             await self.check_response(response)
             return await response.json()
 
+    async def __delete(self, endpoint: str) -> Any:
+        """ Make a POST request to the TaHoma API """
+        async with self.session.delete(f"{self.api_url}{endpoint}") as response:
+            await self.check_response(response)
+            return await response.json()
```

Comment:
The DELETE command does not return anything.

Reply:
@tetienne but it can return a status code in the range of 400/500? That's why I kept the `check_response` check.

Reply:
Sorry, I missed that you pointed to this line `return await response.json()`... DELETE responses can return a JSON response, however in this case they don't. Should I remove it or keep it for other (potential) use-cases.

Reply:
I suggest you to call raise_for_status: https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.ClientResponse.raise_for_status and return nothing.

Reply:
@tetienne but in that case we won't be able to throw exceptions like 'Unauthorized' etc? I would prefer to keep `await self.check_response(response)`, but I can remove `return await response.json()` for now?

Reply:
Indeed, remove `             return await response.json() ` will be enough.